### PR TITLE
Fixes #355, #353

### PR DESCRIPTION
**Pull request by dmpe**
Hi @hadley 
Given that you already pushed some commits to the master and given my GIT (un)knowledge, it is easier for me to open a new PR. 

To that:

Yes, it is not necessary if you mean this line
```
Rscript -e "library(stringi); stri_install_icudt()"
```
There are deprecated. I squashed the commits.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,30 +1,23 @@
...
 
-before_install:
-  - curl -OL http://raw.github.com/craigcitro/r-travis/master/scripts/travis-tool.sh
-  - chmod 755 ./travis-tool.sh
-  - ./travis-tool.sh bootstrap
-install:
-  - ./travis-tool.sh install_deps
-  - sudo Rscript -e "library(stringi); stri_install_icudt()"
-# Configuration options below this line are suggested options.
+r_github_packages:
```
Comment (hadley) [inline .travis.yml]: Does it need to be installed from github?

File: .travis.yml [hunk 2]
```diff
@@ -1,30 +1,23 @@
...
+  - Rexamine/stringi
+
 notifications:
```
Comment (hadley) [inline .travis.yml]: You can just delete this now

File: .travis.yml [hunk 3]
```diff
@@ -1,30 +1,23 @@
...
 
-language: c
-# To build on OSX, switch the previous line to
-# language: objective-c
-script: ./travis-tool.sh run_tests
+language: r
+sudo: required
+warnings_are_errors: true
 
 env:
```
Comment (hadley) [inline .travis.yml]: And this.  Hmmm, might be just as easy to do this myself!

Comment (dmpe) [inline .travis.yml]: yeeh, feel free to do so..... not going to boot up the VM

**Pull request closed without merging**